web/crux-ui/src/pages/api/templates/[templateId]/image.ts

```diff
@@ -0,0 +1,15 @@
+import crux from '@server/crux/crux'
+import { withMiddlewares } from '@server/middlewares'
+import { NextApiRequest, NextApiResponse } from 'next'
+
+const onGet = async (req: NextApiRequest, res: NextApiResponse) => {
+  const templateId = req.query.templateId as string
+
+  const data = await crux(req).templates.getImage(templateId)
+
+  res.status(200).setHeader('Content-Type', 'image/jpeg').send(data.data)
```

Comment:
it think u can get rid of this whole endpoint with next13's custom imageLoader
https://nextjs.org/docs/api-reference/next/image

Reply:
ImageLoader also solves the SSR error handling issue. 🚀